examples/variational_autoencoder.py

```diff
@@ -40,8 +37,8 @@ def sampling(args):
 x_decoded_mean = decoder_mean(h_decoded)
 
 def vae_loss(x, x_decoded_mean):
-    xent_loss = objectives.binary_crossentropy(x, x_decoded_mean)
-    kl_loss = - 0.5 * K.mean(1 + z_log_std - K.square(z_mean) - K.exp(z_log_std), axis=-1)
+    xent_loss = original_dim * objectives.binary_crossentropy(x, x_decoded_mean)
```

Comment:
I'm a bit confused here.
If T is the training set, T_i is the single i-th sample (image) from the training set (with a dim of 784) and t_j is the j-th pixel then when we consider p(x_i | z_i,l) (expr. 10), do we assume that x_i is a t_j (pixel) or T_i (image)?

But I could well be wrong, as I'm no expert in this field, I've started to look into VAE just recently.